transformer_engine/jax/module.py

```diff
@@ -103,14 +103,14 @@ class Softmax(nn.Module):
     Parameters
     ----------
     scale_factor : float, default = 1.0
-        scale the inputs along the last dimension before running softmax.
+        Scale the inputs along the last dimension before running softmax.
```

Comment:
Is this scaling only across the last dimension? The whole softmax input is scaled, right?

Reply:
Yes, it is.